src/utils/query-nonce-meta-tag-content.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Helper method for querying a `meta` tag that contains a nonce value
+ * out of a DOM's head.
+ *
+ * @param doc The DOM containing the `head` to query against
+ * @returns The content of the meta tag representing the nonce value, `undefined` if no tag exists, or
+ * `null` if the tag has no content.
```

Comment:
is there value for the user in having a distinction in the return value between the situation where the tag exists with no content and the case where it isn't there at all? perhaps so, but if we don't think so it might be good for us to just coerce down to a single nullish value with a `?? undefined` (or `?? null`) at the end